portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,116 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>get to know Dez</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <img src="/desmondsimages/headerpic.png" 
+             alt="Dez Image #1" 
+             style="float:left;"
+              class="image1">
+        <h1>
+             get to know Dez
+        </h1>
+            <p class="leftpad1">
+                <strong>NAME</strong>: Desmond Talton <br><br>
+                <strong>COLLEGE</strong>: The Georgia Institute of Technology <br><br>
+                <strong>MAJOR</strong>: Computer Science <br><br>
+            </p>
+            <button onclick="funFactGenerator()">Random Fun Fact</button>
+            <div id="factgetter">
+                Press to Generate a Fun Fact.
+            </div>    
+        <h>
+            <br><br>
+            <img src="/desmondsimages/dezinsuit.jpg" 
+            alt="Dez Image #2" 
+            style="float:right;"
+            class="image2">
+            "Hay, my name ih Dehmun Taltun"
+        </h>
+            <p> 
+                <strong> I've been in college at Georgia Tech, I've been teased for my dialect. </strong><br><br>
+
+                I am from Macon, Georgia (which is a city, but I guess its the outback country in the middle of nowhere) and 
+                that's just how we talk down here. <br><br>
+
+                From the slang and "lingo" I employ to the southern drawl I deliver it with, the way I talk is just 
+                something that makes me... me. In my opinion, the way I talk is the best way to go about communicating.
+                <br><br><br><br>
```

Comment:
(Optional) Consider using css to add margins:
html:
`<p class="description">xxxxx</p>`
css:
```
.description {
 margin-bottom: 20px
}
```

Reply:
Done. I got rid of the <br> tags and replaced them with the CSS formatting with a margin.